### PR TITLE
Update logo in readme

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

**Comment by rekpero** [comment]
Update the package readme as well - storage and cli @aayushmahapatra

File: packages/storage/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <p align="center">
   <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="https://github.com/spheronFdn/sdk/blob/main/.github/assets/logo-dark.svg">
-    <source media="(prefers-color-scheme: light)" srcset="https://github.com/spheronFdn/sdk/blob/main/.github/assets/logo.svg">
-    <img alt="Spheron" src="https://github.com/spheronFdn/sdk/blob/main/.github/assets/logo.svg" width="250">
+    <source media="(prefers-color-scheme: dark)" srcset="https://github.com/spheronFdn/sdk/blob/main/.github/assets/spheron-logo-dark.svg">
```
Comment (rekpero) [inline packages/storage/README.md:3]: update the versions on both the package

**Review by rekpero — CHANGES_REQUESTED** [review]

File: packages/storage/README.md (hunk 1 above)
Comment (aayushmahapatra) [reply packages/storage/README.md:3]: Done.

**Review by rekpero — CHANGES_REQUESTED** [review]
bro change in the package.json file as well

**Review by shomix — APPROVED** [review]

**Review by rekpero — APPROVED** [review]

**Pull request merged**